src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryHelper.js

```diff
@@ -83,6 +83,21 @@
      * @description: invoke the Data Importer on all records associated with this Batch
      */
     processBatch: function(component) {
+        var totals = component.get('v.totals');
+        var record = component.get('v.record');
+        if (record.RequireTotalMatch__c) {
```

Comment:
This should match against the label/name for RequireTotalMatch, otherwise this doesn't work in namespaced orgs.